modules/i18n/src/translations/en-US/portals/applications.ts

```diff
@@ -1672,6 +1690,12 @@ export const applications: ApplicationsNS = {
                                     instruction: "The introspection responses for application access tokens no longer "
                                         + "return the <1>username</1> attribute."
                                 }
+                            },
+                            version200: {
+                                addAllRequestedClaimsInJWTAccessToken: {
+                                    instruction: "Irrespective of the <1>scopes</1> requested, all the <3>Requested Attributes</3> will "
+                                        + "be included in the JWT Access Token."
+                                }
```

Comment:
No need to break into multiple lines.
```suggestion
                                    instruction: "Irrespective of the <1>scopes</1> requested, all the <3>Requested Attributes</3> will be included in the JWT Access Token."
                                }
```

Reply:
Wouldn't it reduce the readability of the sentence?

Reply:
In FE i18n file, moving forward we do not line break.
Line breaks will make it difficult to search for a string and right away translate to a new language.
Please check other places as well.